roles/dtc/common/templates/ndfc_create_fabric.j2

```diff
@@ -145,6 +145,7 @@
 {% endif %}
   ISIS_P2P_ENABLE: {{ underlay.isis.network_point_to_point | default(defaults.vxlan.underlay.isis.network_point_to_point) }}
 {% if (underlay.bfd.enable | default(defaults.vxlan.underlay.bfd.enable) | title) == 'True' %}
+  ISIS_AUTH_KEY: {{ underlay.isis.authentication_key }}
```

Comment:
Do we need a default value?

Reply:
No default expected. Customer should provided their own key. If key is not provided and auth is enabled, syntax validation  will fail.

I moved  Key in the if condition.